packages/flutter_tools/test/asset_bundle_package_test.dart

```diff
@@ -93,8 +93,8 @@ $assetsSection
     }
   }
 
-  group('AssetBundle assets from package', () {
-    testUsingContext('One package with no assets', () async {
+  group('AssetBundle assets from packages', () {
+    testUsingContext('No assets are bundled in case of a package without assets', () async {
```

Comment:
[nit] You might be able to make test names shorter using the scheme `[This happens] when package [is so]` instead of `[This happens] in case of a package with [some condition]`. E.g. `'No assets are bundled when package has no assets'` or (next one) `'One asset is bundled when package lists one asset in the pubspec'`.